lib/sms.js

```diff
@@ -0,0 +1,86 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+var Nexmo = require('nexmo')
+var P = require('bluebird')
+
+var TEMPLATE_NAMES = new Map([
+  [ 1, 'installFirefox' ]
+])
+
+module.exports = function (log, translator, templates, smsConfig) {
+  var nexmo = new Nexmo({
+    apiKey: smsConfig.apiKey,
+    apiSecret: smsConfig.apiSecret
+  })
+  var sendSms = P.promisify(nexmo.message.sendSms, { context: nexmo.message })
+  var firefoxAppStoreLink = smsConfig.firefoxAppStoreLink
+
+  return {
+    send: function (phoneNumber, senderId, messageId, acceptLanguage) {
+      log.info({
+        op: 'sms.send',
+        senderId: senderId,
+        messageId: messageId,
```

Comment:
I wonder if it might be useful to do a similar logging pattern as the mailer. On send, you have a [trace](https://github.com/mozilla/fxa-auth-mailer/blob/master/mailer.js#L215) level log.